src/test/java/uk/ac/ebi/ampt2d/metadata/MetadataApplicationTest.java

```diff
@@ -236,280 +236,240 @@ public void postWebResource() throws Exception {
     }
 
     @Test
-    public void findAssemblyByName() throws Exception {
+    public void findAssemblies() throws Exception {
         String grch37Url = postTestAssembly("GRCh37", "p2",
                 Arrays.asList("GCA_000001405.3", "GCF_000001405.14"));
         String grch38Url = postTestAssembly("GRCh38", "p2",
                 Arrays.asList("GCA_000001405.17", "GCF_000001405.28"));
 
-        mockMvc.perform(get("/assemblies/search/findByName?name=GRCh37"))
+        mockMvc.perform(get("/assemblies?name=GRCh37"))
```

Comment:
If the URL mapping in the controller is `/assemblies/search/`, how does this work?

Reply:
Both `/assemblies?name=GRCh37` and `/assemblies/search?name=GRCh37` would work the same. I think `/assemblies?name=GRCh37` is the default mapping provided. `/assemblies/search?name=GRCh37` is the one specifically defined in the controller.